frontend/src/app/pages/dashboards/organization/[id]/inventory/page.tsx

```diff
@@ -0,0 +1,447 @@
+"use client";
```

Comment:
you need to re-design all the code in this file, use form submit, use one confirm modal, use one change quantity handler, and etc...